docs/root/start/sandboxes/postgres.rst

```diff
@@ -0,0 +1,134 @@
+.. _install_sandboxes_postgres:
+
+Postgres Filter
+===============
+
+In this example, we show how the :ref:`Postgres filter <config_network_filters_postgres_proxy>` 
+can be used with the Envoy proxy. The Envoy proxy configuration includes a Postgres filter that
+parses queries and collects Postgres-specific metrics.
+
+
+Running the Sandboxes
+~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: _include/docker-env-setup.rst
+
+
+Step 3: Build the sandbox
+*************************
+
+.. code-block:: console
+
+  $ pwd
+  envoy/examples/postgres
+  $ docker-compose pull
+  $ docker-compose up --build -d
+  $ docker-compose ps
+
+         Name                      Command               State                             Ports                          
+  ------------------------------------------------------------------------------------------------------------------------
+  postgres_postgres_1   docker-entrypoint.sh postgres    Up      0.0.0.0:5432->5432/tcp                                   
+  postgres_proxy_1      /docker-entrypoint.sh /usr ...   Up      10000/tcp, 0.0.0.0:1999->1999/tcp, 0.0.0.0:8001->8001/tcp
```

Comment:
the formatter needs a single space after `...` see here https://github.com/envoyproxy/envoy/commit/25d87ca0d1c98ba97d4923ebaced643a113c473d

Reply:
Fixed!

Reply:
> Where exactly?

here

Reply:
Ah I fixed it already! Thanks!